src/lib/sections/Navigation/Navigation.stories.tsx

```diff
@@ -73,21 +73,21 @@ const meta: Meta<typeof Navigation> = {
           <Navigation.Content>
             <Navigation.List>
               <Navigation.Item>
-                <Navigation.Link to="#" current="page">
+                <Navigation.Link href="#" aria-current="page">
                   <Navigation.Icon name="information" />
                   <Navigation.Label>Settings</Navigation.Label>
                 </Navigation.Link>
               </Navigation.Item>
             </Navigation.List>
             <Navigation.List>
               <Navigation.Item>
-                <Navigation.Link to="#">
+                <Navigation.Link href="#">
                   <Navigation.Icon name="user" />
                   <Navigation.Label>admin</Navigation.Label>
                 </Navigation.Link>
               </Navigation.Item>
               <Navigation.Item>
-                <Navigation.Link to="#">
+                <Navigation.Link as="button">
```

Comment:
You can do this by importing the Button component from `@canonical/react-components`

```suggestion
                <Navigation.Link as={Button}>
```

Reply:
in the rendered code example part it comes off as `<Navigation.Link as={() => {}}>`, let me give it another try